presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/presto/etc/access-control.properties

```diff
@@ -0,0 +1,2 @@
+# This file is not used.
```

Comment:
This file _is used_, implicitly: https://github.com/prestosql/presto/blob/0039dd8b2922df2c2ce57ccd337c357e5136bc3f/presto-main/src/main/java/io/prestosql/security/AccessControlManager.java#L136